src/JobInterface.php

```diff
@@ -64,4 +64,21 @@ public function sendData($data);
      * @return Promise
      */
     public function sendWarning($warning);
+
+    /**
+     * Sends the result data and marks job as completed
+     *
+     * @param null $data
+     * @return mixed
+     */
+    public function complete($data = null);
+
+    /**
+     * Fails the job with optional error message
+     *
+     * @param null $error
+     * @return mixed
+     */
+    public function fail($error = null);
+
```

Comment:
![image](https://user-images.githubusercontent.com/225374/30540661-5fead9e2-9c78-11e7-8897-ecca22d8044f.png)

The CS check doesn't like that last newline before the closing brace.  ;-)